src/styles/utils/reset.scss

```diff
@@ -0,0 +1,22 @@
+body,
+div,
+h2,
+p {
+  margin: 0;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
```

Comment:
don't need this stuff along with the element,
yet you've learned to hard-hide an element 🙂 
```suggestion
```